interfaces/automated-testing/scd/scd.yaml

```diff
@@ -35,6 +35,402 @@ components:
+        stored privately by the DSS and then compared against entries in a Key
+        provided to mutate the airspace.  The EntityOVN is also provided by the
+        client whenever that client transmits the full information of the Entity
+        (either via GET, or via a subscription notification).
+      type: string
+      minLength: 16
+      maxLength: 128
+      example: 9d158f59-80b7-4c11-9c0c-8a2b4d936b2d
+    SubscriptionID:
+      description: >-
+        Identifier for a subscription communicated through the DSS.  Formatted
+        as a UUIDv4.
+      anyOf:
+        - $ref: "#/components/schemas/UUIDv4Format"
+      example: 78ea3fe8-71c2-4f5c-9b44-9c02f5563c6f
+
+
+    OperationalIntentUssBaseURL:
+      description: >-
+        The base URL of a USS implementation that implements the parts of the
+        USS-USS API necessary for providing the details of this operational
+        intent, and telemetry during non-conformance or contingency, if
+        applicable.
+      anyOf:
+        - $ref: "#/components/schemas/UssBaseURL"
+
+    OperationalIntentState:
+      description: >-
+        State of an operational intent.
+        'Accepted': Operational intent is created and shared, but not yet in
+        use; see standard text for more details.
+        The create or update request for this operational intent reference must
+        include a Key containing all OVNs for all relevant Entities.
+        'Activated': Operational intent is in active use; see standard text for
+        more details.
+        The create or update request for this operational intent reference must
+        include a Key containing all OVNs for all relevant Entities.
+        'Nonconforming': UA is temporarily outside its volumes, but the
+        situation is expected to be recoverable; see standard text for more
+        details.
+        In this state, the `/uss/v1/operational_intents/{entityid}/telemetry`
+        USS-USS endpoint should respond, if available, to queries from USS
+        peers.  The create or update request for this operational intent may
+        omit a Key in this case because the operational intent is being adjusted
+        as flown and cannot necessarily deconflict.
+        'Contingent': UA is considered unrecoverably unable to conform with its
+        coordinate operational intent; see standard text for more details.
+        This state must transition to Ended.  In this state, the
+        `/uss/v1/operational_intents/{entityid}/telemetry` USS-USS endpoint
+        should respond, if available, to queries from USS peers.  The create or
+        update request for this operational intent may omit a Key in this case
+        because the operational intent is being adjusted as flown and cannot
+        necessarily deconflict.
+      type: string
+      enum:
+      - Accepted
+      - Activated
+      - Nonconforming
+      - Contingent
+    Key:
```

Comment:
This schema is not used in this interface. It can be removed.

Reply:
Thanks for this, removing now.